contrib/odata-hierarchies-2-sql/sqlite3/traverse-example-section-3.3.sql

```diff
@@ -0,0 +1,98 @@
+.read "create-schema.sql"
+
+-- logic for a depth first search traversal of a hierarchy
+with recursive
+  -- traversal needs a stable sibling order; can be rowid if selecting from hierarchy table, or row_number() if preceding transformation
+  salesorg_w_sibling(ID,Superordinate,Name,SiblingIdx) as (select *,rowid from SalesOrganization),
+  parent_of(node,parent_node) as (select id, Superordinate from salesorg_w_sibling),
+  dfs_traversal_of(node, path) as (
+    select id, '0001' from salesorg_w_sibling -- start with root node
```

Comment:
This assumes that there is only one root node and that that node is getting the rowId 1. 
Might be better to use the SiblingIdx column from salesorg_w_sibling instead of the hardcoded '0001'
(see also comment below about moving the conversing to four digits into salesorg_w_sibling